### ISSUE TITLE
Elasticsearch - ordering by relevance

### ISSUE THREAD

**Issue by vitek-rostislav** [shopsys/shopsys#2165]
### Is your feature request related to a problem?
I am not sure how the Elasticsearch results are ordered by default. It is kind of confusing that [here](https://github.com/shopsys/shopsys/blob/master/packages/framework/src/Model/Product/Search/FilterQuery.php#L65), all the ordering modes are implemented, but `ProductListOrderingConfig::ORDER_BY_RELEVANCE` is omitted. **If this is the mode that is used by default by Elastic, it would be nice to mention this fact in the annotation of the method so it is clear for the project developers**, and moreover, they know they need to add the following condition when they want to modify the relevance ordering behavior:
```
if ($orderingModeId === ProductListOrderingConfig::ORDER_BY_RELEVANCE) { ... }
```
**It would be nice to examine the default Elasticsearch ordering - if it differs from the expected relevance, the above condition should be added to `FilterQuery::applyOrdering` method**

**Comment by grossmannmartin** [comment]
Hi @vitek-rostislav,

Thanks for filling the issue!

I did look at the default search ordering in Elasticsearch and it's by default sorted by relevance (https://www.elastic.co/guide/en/elasticsearch/guide/current/sorting.html) and the relevance is represented by the field `_score` (https://www.elastic.co/guide/en/elasticsearch/guide/current/relevance-intro.html).

So comment/annotation in the class seems to me like a nice thing to do to keep future generations from the same confusion as we had 😄

**Comment by grossmannmartin** [comment]
fixed by #2194
